### PR TITLE
Update money and monetize gems to latest versions

### PR DESCRIPTION
**Pull request by npezza93**

### CODE REVIEW

File: money-rails.gemspec
Comment (semmons99) [inline money-rails.gemspec:30]: @antstorm any issue with locking to a less rigid version here?

Comment (npezza93) [reply money-rails.gemspec:30]: None from me! What do you think of `< 2.0`?

Comment (npezza93) [reply money-rails.gemspec:30]: Also should `money` be loosened to `< 7.0` as well?

Comment (semmons99) [reply money-rails.gemspec:30]: it should be okay, but it would be a change of policy. I am cool with it if @antstorm cannot think of any adverse consequences

File: money-rails.gemspec
Comment (stim371) [inline money-rails.gemspec:29]: @semmons99 the `<` constraint seems super loose (versus `~> major.minor` on the current version). would this gem really function with v1 of `money` and v0.x of `monetize`?

Comment (semmons99) [reply money-rails.gemspec:29]: good catch. we should probably just make it `~> 6.13` right?

File: CHANGELOG.md
Comment (semmons99) [inline CHANGELOG.md:7]: ~> 6.13

File: CHANGELOG.md
Comment (semmons99) [inline CHANGELOG.md:8]: ~> 1.something

**Review by semmons99 — CHANGES_REQUESTED** [review]

**Review by semmons99 — APPROVED** [review]

**Comment by swrobel** [comment]
@semmons99 can we get a new release with this included? Thanks!

**Comment by semmons99** [comment]
sure, just waiting for an approval on the version bump or I just created

**Comment by semmons99** [comment]
@swrobel released

**Pull request merged**
